docker/cache/resources/repo-config.ttl

```diff
@@ -22,8 +22,12 @@
             owlim:enable-context-index "false" ;
 
             # indexes to speed up the read queries
-            owlim:enablePredicateList "true" ;
-            owlim:enable-literal-index "true" ;
-            owlim:in-memory-literal-properties "true" ;
+#            owlim:enablePredicateList "true" ;
+#            owlim:enable-literal-index "true" ;
+#            owlim:in-memory-literal-properties "true" ;
+            owlim:enablePredicateList "false" ;
```

Comment:
what does this disable?

Reply:
These are recommended settings from here 
http://graphdb.ontotext.com/documentation/standard/data-loading-query-optimisations.html

Reply:
However they should probably / possibly be switched back on subsequent to load and possibly prior to the post processing rdf queries. For post processing I'm not sure that we are seeing a big performance hit with them switched off though.